packages/@cdklabs/typewriter/src/documented.ts

```diff
@@ -85,6 +85,12 @@ export interface DocsSpec {
    * @default none
    */
   see?: string;
+  /**
+   * The `@cloudformationResource` string for Cfn resource classes
+   *
+   * @default none
+   */
+  cloudformationResource?: string;
```

Comment:
`@cdklabs/typewriter` is intended as a general purpose tool. The other doc flags are all universally applicable and/or inspired by jsii which also aims to be a universal tool..

I think we need a generic feature to support arbitrary additional doctags.